webapp/default.conf

```diff
@@ -0,0 +1,10 @@
+server {
+    listen       80 default_server;
+    server_name  localhost;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html;
+        try_files $uri /index.html;
```

Comment:
This addresses #292 by returning `index.html` if the requested path isn't an existing file.

Reply:
To clarify: This copies the default config from webapp and sets nginx config through the docker build? Previously, was nginx config set without the try_files line?

Reply:
Previously the `nginx` service was forwarding requests for the webapp in production to Netlify using `proxy_pass`.

Now, instead of building on Netlify, the webapp gets built and deployed in an nginx container on the swarm that everything else runs in.

It would have been possible to set up redirect rules in Netlify to behave the same as `try_files` here, but I'm trying to reduce the number of technologies maintainers need to know to work on the infrastructure.

Does that clarify things?

Reply:
Yes it does. Thank you so much and the rest of the PR was well explained.